Suggested Logo for Edam Ontology

Babsbarokah opened the pull request:
EDAM Ontology Logo creation(PROCESS) 


EDAM is a domain ontology of data analysis and data management in life sciences (biology, medicine, ecology, etc) 

Why the Logo? 

A new logo is needed for re-branding and to portray brand identity. 

Brand Specialty &Value:
Bring expert in various domains of science to lower the barrier and effort towards doing scientific research in a more OPEN, RELIABLE, and INCLUSIVE way.

Target Audience:
 Professionals, amateur, academic, industrial, and citizen scientists

Competition:
Did research on common and existing BIO scientific website in the industry and was able to highlight a few

Design style:
Retro& Modern (History is important and whatever info given is right) 

Moodboard: 
Classification tree, DNA molecule, Alphabet E

Attention to colour
Blue    - Trust& Productive
Green  -Reliable 
White  - Minimal

~Analogous colours


The overall idea behind the logo and process is to create a simple, timeless, memorable, and distinctive brand identity, where it can be used anywhere and as any size. 

However, it still needs iteration, I made a pull request to the current design. Looking forward to your review.

Thanks

Babsbarokah committed:
Suggested Logo for Edam Ontology

matuskalas reviewed (commented):
Looks NICE, thank you @Babsbarokah!

Should I merge it, or do you want to add any changes? (you can also mark|unmark the PR as draft|ready)

Babsbarokah commented:
Thanks.

Yes, currently I have no update to this.

bryan-brancotte commented:
Hi @Babsbarokah 

It looks really good :clap: :clap: I like it !

As @matuskalas talked about square and rectangle rendering in https://github.com/edamontology/edamontology/issues/666, I took the liberty to try some rendering in the attached svg files. here are some low res export to have an quick idea.


![rectangle](https://user-images.githubusercontent.com/11556772/116414739-037c1d80-a839-11eb-8d90-bd0056f6660f.png)
![square](https://user-images.githubusercontent.com/11556772/116414743-0414b400-a839-11eb-8df5-39d338c3a7d1.png)

[EDAM Logo by babsbarokah plus brancotte suggestion for square en rectangle presentation.zip](https://github.com/edamontology/logoi/files/6392212/EDAM.Logo.by.babsbarokah.plus.brancotte.suggestion.for.square.en.rectangle.presentation.zip)

Any remarks, objections, and comments are welcome :)

Babsbarokah commented:
Okayyyyy, I get a clearer view now
The logo needs to fit the highlighted in issue #666

Your square&rectangle rendering looks nice.

Alignment&text needs some edit.
I'll work on the  draft, thanks.

Babsbarokah commented:
I've updated the file, kindly check.
![Edited logo by babsbarokah](https://user-images.githubusercontent.com/81495038/116551389-cb83e180-a8ef-11eb-8a33-1ec305a5c037.png)

Babsbarokah commented:
> I've updated the file, kindly check.
> ![Edited logo by babsbarokah](https://user-images.githubusercontent.com/81495038/116551389-cb83e180-a8ef-11eb-8a33-1ec305a5c037.png)



> I've updated the file, kindly check.
> ![Edited logo by babsbarokah](https://user-images.githubusercontent.com/81495038/116551389-cb83e180-a8ef-11eb-8a33-1ec305a5c037.png)



> I've updated the file, kindly check.
> ![Edited logo by babsbarokah](https://user-images.githubusercontent.com/81495038/116551389-cb83e180-a8ef-11eb-8a33-1ec305a5c037.png)

1&2 as rectangle and 3 as square

bryan-brancotte commented:
Hi, Thanks for the updates !

~~I think you meant `square` for the fourth one, not Horizontal.~~ you indicated it later

(I like the first "Horizontal", and the triangle too.)

Could you use a fixed width font when EDAM is vertically written ? As a UI/UX Designer (which I am not) do you think it is possible, and/or not a bad idea to manage to aligne Ontology in the Square to the left of the drawing, and the right of "EDAM" ?

Babsbarokah commented:
Yes, I realized I uploaded a slightly different file, decided to quote instead of deleting, since its still a work in progress.

The sizes of the font are/need to be fixed for consistency, I guess its the font type that made it come out like that. I could work on finding a better type face. 

There is no such thing like a bad idea, it only create room for creativity. By aligning to the left and right of the drawing and "EDAM" respectively do you mean center alignment? Maybe if you try to illustrate I'd get it better.

bryan-brancotte commented:
Hi @Babsbarokah 

Thanks for you message, here is the alignment in red that I am talking about, I feel that the elements of the square logo should be inside a virtual square (almost square, here it is a rectangle) defined by the alignment you see in red there.
![square](https://user-images.githubusercontent.com/11556772/116679518-e6675c00-a9aa-11eb-99df-8966e444635e.png)

Babsbarokah commented:
Okay, thank you.

I'd work on this.

Babsbarokah commented:
Yes, updated. Thanks for ur review @bryan-brancotte. Also I'm including for other sizes because their might be some places it will come in handy now/ in future.  Edited the typo. @matuskalas @bryan-brancotte  Do you think it need any further touch?
![Edited logo by babsbarokah (1)](https://user-images.githubusercontent.com/81495038/116829616-87693900-ab9c-11eb-88af-b82345daedc9.png)

Babsbarokah committed:
Suggested EDAM l
